test/jsonrpc_advanced_test.js

```diff
@@ -133,6 +125,114 @@ describe('jsonrpc_advanced',()=>{
       });
     });
   });
+
+  it('#3 Should computeTask and GetTaskResult endpoint', async function(){
+    if(!tree['all'] || !tree['#3']){
+      this.skip();
+    }
+    return new Promise(async resolve => {
+      // create all the boring stuff
+      const principalServer = principalMock.create(getStateKeysCallback);
+      await testUtils.sleep(150);
+      const principalPort = principalMock.getPort(principalServer);
+      const uri = 'http://127.0.0.1:';
+
+      let {bNode,peer} = await testBuilder.createTwo({
+        bOpts:{withProxy : true,proxyPort : 3346,withLogger : false},
+        pOpts :{withLogger : false, principalUri :uri + principalPort}});
+
+      await testUtils.sleep(3000);
+      let bNodeController = bNode.mainController;
+      let bNodeCoreServer = bNode.coreServer;
+      let peerController = peer.mainController;
+      let peerCoreServer = peer.coreServer;
+      // stop the test
+      const stopTest = async ()=>{
+        await peerController.shutdownSystem();
+        peerCoreServer.disconnect();
+        await bNodeController.shutdownSystem();
+        bNodeCoreServer.disconnect();
+        principalMock.destroy(principalServer);
+        resolve();
+      };
+      const client = jayson.client.http('http://localhost:3346');
+      let signKey = await peerController.getNode().selfSubscribeAction();
+      await testUtils.sleep(1000);
+      const computeInput = await getComputeRequest(signKey);
+      client.request('sendTaskInput',computeInput,async (err,res)=>{
+        assert.strictEqual(true, res.result.sendTaskResult, "sendTaskResult not true");
+        await testUtils.sleep(1000);
+        client.request('getTaskResult',
+          {"taskId":computeInput.taskId},
+          async (err,res)=>{
+            if(err) assert.strictEqual(true,false,"err" + err);
```

Comment:
Is this the way to send an error? doesn't look good

Reply:
This will crash the UT